src/components/ParticipantInfo/AudioTrackPublicationInfo/AudioTrackPublicationInfo.tsx

```diff
@@ -44,7 +45,7 @@ export const AudioTrackInfo: React.FC<{
           <Datum label="Codec" value={trackData.codec} />
           <Datum label="Jitter" value={trackData.jitter} />
           <Datum label="Packets Lost" value={trackData.packetsLost} />
-          <Datum label="Packet Loss Percentage" value={lossPercentage! + '%'} />
+          <Datum label="Packet Loss Percentage" value={lossPercentage!} />
```

Comment:
I think the `!` doesn't feel right, since it's suppressing an error about the variable being used before its assigned. I think we can get rid of the `!` if we give `lossPercentage` a sensible default value above:

`let lossPercentage: string | null = '';`